docker-compose.yml

```diff
@@ -19,7 +20,7 @@ services:
       - mysql
       - db_phpunit
     ports:
-      - "80:80"
+      - "8080:80"
```

Comment:
Port 80 shouldn't cause conflicts with existing services when running inside the Vagrant wrapper since it uses a dedicated network interface. The documentation and instructions still use the Vagrant wrapper so maybe we can adjust this later if needed.